assets/css/main.css

```diff
@@ -123,7 +123,7 @@ img {
 }*/
 
 #case-study img {
-  max-width: 50vw;
+  max-width: 33vw;
```

Comment:
This is good. However it is small on mobile devices. Could we include a media query to set the max width value so the pictures are wider on mobile devices but smaller on large screens?

Reply:
Yes, I'll do that.

Reply:
So I don't this change made the images smaller in the mobile view, but I agree that the images do need to be larger when in the mobile view. So I just committed a change that updates this property for all the different media queries. As you update the size of your window the responsive design should yield decently sized diagrams.